src/sdp/SDP.ts

```diff
@@ -13,6 +13,11 @@ export default interface SDP {
    */
   clone(): SDP;
 
+  /**
+   * Retuns SDP string
+   */
+  toString(): string;
```

Comment:
In 2.28.0, this needs to be optional addition as not sure if any builders are implementing SDP 😅. If they are implementing then (highly unlikely) then they would have to implement this method forcibly. In 3.0.0-beta.0 should be ok as we are making breaking change.

Reply:
This would look like below in TS:
```
toString?(): string;
```

Example: https://github.com/aws/amazon-chime-sdk-js/blob/main/src/audiovideoobserver/AudioVideoObserver.ts#L19